actors/user/src/main/java/org/sunbird/user/actors/UserManagementActor.java

```diff
@@ -641,6 +639,21 @@ private void processUserRequest(
     TelemetryUtil.telemetryProcessingCall(userMap, targetObject, correlatedObject);
   }
 
+  private void setStateValidation(Map<String, Object> userMap) {
+    String rootOrgId = (String) userMap.get(JsonKey.ROOT_ORG_ID);
+    String custodianRootOrgId = getCustodianRootOrgId();
+    userMap.put(JsonKey.IS_STATE_VALIDATED, false);
+    if(!rootOrgId.equals(custodianRootOrgId)) {
```

Comment:
avoid these if clauses. They increase branch complexity un-necessarily. Use this instead
`userMap.put(JsonKey.IS_STATE_VALIDATED, !rootOrgId.equals(custodianRootOrgId)`

Reply:
This is minute but i thought it is powerful. Thanks.